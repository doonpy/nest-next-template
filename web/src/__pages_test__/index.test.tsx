import { shallow } from 'enzyme';
import React from 'react';

import Home from '../pages';

describe('index', () => {
  it('should render correctly', () => {
    const component = shallow(<Home />);

    expect(component).toMatchSnapshot();
  });
});
