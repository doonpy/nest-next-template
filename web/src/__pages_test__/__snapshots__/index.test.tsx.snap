// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index should render correctly 1`] = `
<h1>
  Hello, Index!
</h1>
`;
